bin/lhapdf-getdata

```diff
@@ -34,7 +34,8 @@ def getPDFSetList(url):
         logging.debug(pdflistpage)
         hreq.close()
         import re
-        re_anchor = re.compile(r'^\s*<tr>.*<td><a\s+href="([^"]+)">\1.*$')
+##        re_anchor = re.compile(r'^\s*<tr>.*<td><a\s+href="([^"]+)">\1.*$')
+        re_anchor = re.compile(r'^\s*<td\s+class="name"><a.*title="([^"]+)">\1.*$')
```

Comment:
by alisw policy, simply remove the line instead of commenting it out

Reply:
I've removed the commented line.